components/infieldprogresscircle/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@spectrum-css/infieldprogresscircle",
+  "version": "0.0.0",
```

Comment:
Maybe get a second opinion, but should this version number be 1.0.0? Or does that happen when changesets creates a release for us to merge?